build/verify-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+NEXT_RELEASE="$1"
+LAST_RELEASE="$2"
+
+EXIT_CODE=0
+
+# Prints that $FILE is out of sync
+#
+# @param FILE - name of file out of sync
+fail() {
+  local 'WHICH'
+  EXIT_CODE=1
+  WHICH="$1"
+  printf "\033[1;31m[error]\033[0m please run \"npm run release\" %s does not have correct version\n" "$WHICH"
+}
+
+# Runs a command and if the command fails, prints that file is bad.
+#
+# @param FILE - name of file being tested
+# @param ...args - command to run
+try() {
+  local 'FILE'
+  local 'COMMAND'
+
+  FILE="$1"
+  COMMAND=("${@:2}")
+
+  # Execute command, if it fails, exec fail
+  "${COMMAND[@]}" > /dev/null || fail "$FILE"
+}
+
+# Print
+echo "Checking if all files have version: $NEXT_RELEASE"
+
+# Ensure that package.json is in sync
+try "package.json" test "$(jq --raw-output .version package.json)" = "$NEXT_RELEASE"
+try "package-lock.json" test "$(jq --raw-output .version package.json)" = "$NEXT_RELEASE"
```

Comment:
is `jq` on travis already?

Reply:
si, https://docs.travis-ci.com/user/build-environment-updates/2014-12-09/#2014-12-09

Reply:
si, https://travis-ci.com/smartcar/javascript-sdk/builds/88473284#L55